examples/service-clients/azure-client/external-controller/tsconfig.json

```diff
@@ -1,12 +1,8 @@
 {
-	"extends": [
-		"../../../../common/build/build-common/tsconfig.base.json",
-		"../../../../common/build/build-common/tsconfig.cjs.json",
-	],
+	"extends": "../../../../common/build/build-common/tsconfig.node16.json",
 	"exclude": ["dist", "node_modules"],
 	"compilerOptions": {
```

Comment:
exclude "li" as well

Reply:
I removed CJS, so now it's just 'lib' only.  Thx.